conans/util/progress_bar.py

```diff
@@ -43,15 +40,11 @@ def tell(self):
     def read(self, size):
         prev = self.tell()
         ret = self._fileobj.read(size)
-        self._pb.update(self.tell() - prev)
+        self._tqdm_bar.update(self.tell() - prev)
         return ret
 
     def pb_close(self):
-        self._pb.close()
-
-    def pb_write(self, message):
```

Comment:
Not used now, but will be needed soon.

Reply:
When it is used, we'll add it. Do not introduce functionality until it is actually needed.